devicon.json

```
@@ -2774,6 +2774,29 @@
         "color": "#E24329",
         "aliases": []
     },
+    {
+        "name": "gitpod-io",
+        "altnames": [],
```

I suggest changing the name to `gitpod` instead of `gitpod-io`, and instead use `gitpod.io` as an altname
```suggestion
        "name": "gitpod",
        "altnames": ["gitpod.io"],
```

If you agree, the icons and filepaths also need to change to `gitpod`